smk-edit/static/components/edit-tool-details-geomark.html

```diff
@@ -0,0 +1,19 @@
+<div>
+    <edit-tool-details
+        v-bind:toolType="toolType"
+        v-bind:toolInstance="toolInstance"
+    ></edit-tool-details>
+
+    <div class="row">
```

Comment:
I don't think I would expose this in the editing UI. It's more of a 'just in case' property than something a client should be able to change easily.

Reply:
I'd rather leave these properties included. It looks like they're included for all panel tools and excluded for non-panel tools. Removing these means users can't change the ordering, icon, title, and other common panel properties.

Reply:
Hmm, my comment ended up misplaced. I'm referring to the geomark serivce URL config, not the other properties that we want to keep as you mentioned. Sorry for the confusion!

Reply:
Have you decided whether or not to leave in the Geomark service URL field?

Reply:
Thanks for reminding me. Yeah, I'd like to leave it in for now. I'll also get Nicole's feedback and we can remove it in another change as needed.